_dashboards/management/advanced-settings.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: default
+title: Advanced settings
+parent: Dashboards Management
+nav_order: 40
+---
+
+# Advanced settings
+
+Use the **Advanced settings** page to modify settings that govern OpenSearch Dashboards behavior. For example, you can use the page to enable or disable the dark mode, change the default language, change the default number of results displayed on a dashboard, or change the default time zone. See the following sections to learn more about the specific settings.
+
+To modify settings in Advanced Settings, follow these steps:
+
+1. Open the OpenSearch Dashboards main menu, then select **Dashboards Management** > **Advanced settings**.
+2. Search for the setting you want to modify.
+3. Enter a new value for the setting. 
+4. Select **Save changes**.
```

Comment:
Some settings also require and prompt a page refresh after saving, such as theme or dark/light mode change.